deploy/assisted-image-service-service.yaml

```diff
@@ -13,6 +13,7 @@ spec:
       port: 8080
       protocol: TCP
       targetPort: 8080
+      nodePort: 30001
   selector:
     app: assisted-image-service
   type: LoadBalancer
```

Comment:
Do we want nodeport or LoadBalancer?

Reply:
I need a static `nodePort` port, but I still want to get external IP when deployed on AWS/minikube, so I kept it LoadBalancer

Reply:
couldn't we use some templating instead (kustomize/helm) to generate a cleaner manifest?

Reply:
This change shouldn't have any affect in other deployments (not kind) as nodePort was given either way, we would just had to search for it :)

Reply:
the problem I see is that keeping LoadBalancer will make users think something is wrong (pending external IP forever), also defining fixed nodePort for apparently no reason (in the minikube's case) can cause confusion.
I think it'd be cleaner if we define it properly according to each use case